packages/b/bzflag/pspec_x86_64.xml

```diff
@@ -59,7 +59,8 @@
             <Path fileType="library">/usr/lib64/bzflag/wwzones.so</Path>
             <Path fileType="library">/usr/lib64/systemd/system/bzflag.service</Path>
             <Path fileType="library">/usr/lib64/sysusers.d/bzflag.conf</Path>
-            <Path fileType="data">/usr/share/applications/bzflag.desktop</Path>
+            <Path fileType="data">/usr/share/appdata/org.bzflag.BZFlag.appdata.xml</Path>
```

Comment:
Should be `/usr/share/metainfo/` appdata path is deprecated

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-location

Reply:
Even appdata.xml should be on `/usr/share/metainfo`? Then should we move all metadata in `/usr/share/appadata/` to `/usr/share/metainfo/` ? Like `clear-sans` is in appdata for some reason(Did I do it?)